src/Icon/Icon.tsx

```diff
@@ -666,6 +666,203 @@ export const Icon: FC<Props> = ({
         </svg>
       )
     }
+
+    {
+      //CAR MTA
+      render === 'car-mta' && (
+        <svg
+          width="47"
```

Comment:
Change the width and height to 100% (but leave the viewbox the same) and then each icon becomes scaleable!